cypress/integration/registryMigration.spec.js

```diff
@@ -6,7 +6,7 @@ const ENABLED_COLOUR = 'rgb(83, 132, 254)'
 
 describe('Migrate a subdomain to new registry', () => {
   it('can visit an unmigrated name and migrate it', () => {
-    cy.visit(`${ROOT}/name/sub1.testing.eth`)
+    cy.visit(`${ROOT}/name/sub1.testing.eth/details`)
```

Comment:
Is this added just to speed up testing speed, or did you change the behaviour of how routing works?

Reply:
We had some duplicate code, so I am now doing a redirect when landing on that page